### DISCUSSION TITLE
Generated source code includes database name?

### DISCUSSION THREAD

**Discussion by oschrenk** [slick/slick#2285] [Questions]
I use https://github.com/tototoshi/sbt-slick-codegen to generate my `Tables.scala` which I then intend to commit to git.

I noticed that the database name is also part of the generated source code `Some("acme")`.

```
[...]
class SomeTable(_tableTag: Tag) extends profile.api.Table[FlywaySchemaHistoryRow](_tableTag, Some("acme"), "some_table")
[...]
```

Since production code might run against a different database name, is that an issue? Is there some way to not generate the database name? If so, are there any downsides?

**Comment by djx314** [comment]
I suggest that have a choice. Some people want to use the code to product and no need to change the gen code.
But many people want to protect their production database. This is the useful way.

**Comment by nafg** [comment]
What database are you using? That is where the schema should go, not the database name. I think MySQL equates them though.

**Comment by jjmalina** [comment]
I might be having this same issue which started after I upgraded from slick 3.1.1 to 3.3.3: https://gist.github.com/jjmalina/05885a13ea9df29893da8205ce21e0ed
@nafg any idea?

**Reply by nafg** [reply]
Not offhand, but
1. The latest version is 3.5.1
2. If you still need help I could do a call in about a week, DM me on Discord

**Reply by jjmalina** [reply]
I changed the database name I use when running the codegen to match my production database and that solved the issue.

**Comment by nafg** [comment]
Is this an issue with Slick or with [tototoshi/sbt-slick-codegen](https://github.com/tototoshi/sbt-slick-codegen)?

**Reply by jjmalina** [reply]
For me it happened using `slick-codegen`: https://scala-slick.org/doc/stable/code-generation.html#standalone-use
